### DISCUSSION TITLE
Export Data from "ODF along a fibre", or plot various ODFs along one fiber

### DISCUSSION THREAD

**Discussion by PedroPinh** [mtex-toolbox/mtex#1235] [Ask Anything]
Hi. I'm trying to export data from this plot:
![ODF along gamma fiber](https://user-images.githubusercontent.com/91805626/135763716-4976afaa-2035-49fb-82b5-c57b3efb36d8.jpg)
I tryed exporting the ODF data, didn't work. Now I'm trying to export this data. If it is not possible to export this data, I would like to know if it is possible to plot multiple ODFs along just one fiber, like the following graphs:
![ODFs along one fiber](https://user-images.githubusercontent.com/91805626/135763848-0f47c700-4f77-452d-baa9-a3734fdee891.jpeg)
Glad for your answers.

**Comment by ralfHielscher** [comment]
Hi Pedro,

assuming your ODF is stored in a variable |odf| you can evaluate it in any list of orientations |ori| you want using the command
```matlab
odf.eval(ori)
```
E.g. if you want to vary the phi1 angle and fix the other you can to
```matlab
phi1 = linspace(0,90*degree);
Phi = 45*degree;
phi2 = 0*degree;
ori = orientation.byEuler(phi1, Phi, phi2, odf.CS)
plot(phi ./ degree, odf.eval(ori))
```
I hope this helps,

Ralf.
